src/Sylius/Behat/Context/Ui/Admin/ManagingChannelsContext.php

```diff
@@ -343,9 +344,9 @@ public function iShouldSeeChannelsInTheList(int $numberOfChannels = 1): void
     }
 
     /**
-     * @Then the code field should be disabled
+     * @Then I should not be able to edit its code
```

Comment:
```suggestion
     * @Then I should not be able to edit its code
     * @Then the code field should be disabled
```

I know there's no BC policy for Behats, but let's be nice and leave the step in case anyone is using it.